ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -38,19 +41,25 @@ class JsonFeature(val serializer: JsonSerializer) {
          * Default value is [defultSerializer]
          */
         var serializer: JsonSerializer? = null
+        var allowedContentTypes: List<ContentType>? = null
```

Comment:
Can't it be simply `listOf(ContentType.Application.Json)` instead of nullable so there will be no need to have elvis in `prepare`

Reply:
Also it should be prohibited to set an empty list of content types

Reply:
Also this property requires kdoc and `@KtorExperimentalAPI` since it will be deprecated in the future once client-side content negotiation will be introduced.

Reply:
As you pointed out, 
- changed ```allowedContentTypes``` variable to non-nullable type.
- prohibited initialization with empty list of content type, by throwing a proper exception.
- added some documentation for explaining of this config variable.

Thanks for your advice!